test/simple_select.test.js

```diff
@@ -0,0 +1,346 @@
+      map.fire('mousemove', makeMouseEvent(startPosition[0] + 25, startPosition[1] + 25));
+      map.fire('mouseup', makeMouseEvent(startPosition[0] + 25, startPosition[1] + 25));
+
+      var movedPoint = Draw.get(pointId);
+      t.equal(movedPoint.geometry.coordinates[0], startPosition[0] + 25, 'point lng moved');
+      t.equal(movedPoint.geometry.coordinates[1], startPosition[1] + 25, 'point lat moved');
+      t.equal(countPositions(movedPoint), countPositions(cloneFeature('point')), 'point has same number of postions');
+
+      var movedMultiPoint = Draw.get(multiPointId);
+      t.equal(movedMultiPoint.geometry.coordinates[0][0], cloneFeature('multiPoint').geometry.coordinates[0][0] + 25, 'multipoint lng moved');
+      t.equal(movedMultiPoint.geometry.coordinates[0][1], cloneFeature('multiPoint').geometry.coordinates[0][1] + 25, 'multipoint lat moved');
+      t.equal(countPositions(movedMultiPoint), countPositions(cloneFeature('multiPoint')), 'multiPoint has same number of postions');
+
+      var movedLineString = Draw.get(lineStringId);
+      t.equal(movedLineString.geometry.coordinates[0][0], cloneFeature('line').geometry.coordinates[0][0] + 25, 'line lng moved');
+      t.equal(movedLineString.geometry.coordinates[0][1], cloneFeature('line').geometry.coordinates[0][1] + 25, 'line lat moved');
+      t.equal(countPositions(movedLineString), countPositions(cloneFeature('line')), 'line has same number of postions');
+
+      var movedMultiLineString =Draw.get(multiLineStringId);
+      t.equal(movedMultiLineString.geometry.coordinates[0][0][0], cloneFeature('multiLineString').geometry.coordinates[0][0][0] + 25, 'multiLineString lng moved');
+      t.equal(movedMultiLineString.geometry.coordinates[0][0][1], cloneFeature('multiLineString').geometry.coordinates[0][0][1] + 25, 'multiLineString lat moved');
+      t.equal(countPositions(movedMultiLineString), countPositions(cloneFeature('multiLineString')), 'multiLineString has same number of postions');
+
+      var movedPolygon =Draw.get(polygonId);
+      t.equal(movedPolygon.geometry.coordinates[0][0][0], cloneFeature('polygon').geometry.coordinates[0][0][0] + 25, 'polygon lng moved');
+      t.equal(movedPolygon.geometry.coordinates[0][0][1], cloneFeature('polygon').geometry.coordinates[0][0][1] + 25, 'polygon lat moved');
+      t.equal(countPositions(movedPolygon), countPositions(cloneFeature('polygon')), 'polygon has same number of postions');
+
+      var movedMultiPolygon =Draw.get(multiPolygonId);
+      t.equal(movedMultiPolygon.geometry.coordinates[0][0][0][0], cloneFeature('multiPolygon').geometry.coordinates[0][0][0][0] + 25, 'multiPolygon lng moved');
+      t.equal(movedMultiPolygon.geometry.coordinates[0][0][0][1], cloneFeature('multiPolygon').geometry.coordinates[0][0][0][1] + 25, 'multiPolygon lat moved');
+      t.equal(countPositions(movedMultiPolygon), countPositions(cloneFeature('multiPolygon')), 'multiPolygon has same number of postions');
```

Comment:
Yep. Didn't realize there was an issue for it. Will add it to the PR so it closes when merged.